bundles/org.openhab.binding.bluetooth.bluez/README.md

```diff
@@ -61,3 +62,10 @@ This is how an BlueZ adapter can be configured textually in a *.things file:
 ```
 Bridge bluetooth:bluez:hci0 [ address="12:34:56:78:90:AB", backgroundDiscovery=false ]
 ```
+
+##Troubleshooting
```

Comment:
This should be removed, since we define that bundle to be part of the feature. So when it is correctly installed, the bundle will be there, no question.

Reply:
done